README.md

```diff
@@ -149,10 +146,14 @@ Guide to build images again with modification or with different environment vari
 
 Now you'll need to do steps (2) everytime you make a changes and want to build the images again.
 
-**Note:**
+#### Notes:
    - Default username & password are `admin`.
    - Default domains are: dashboard.openwisp.org, controller.openwisp.org, radius.openwisp.org and topology.openwisp.org.
-   - You may want to add the domains in your hosts file, command: `echo "127.0.0.1 dashboard.openwisp.org controller.openwisp.org radius.openwisp.org topology.openwisp.org" >> /etc/hosts/`
+   - `To reach the dashboard` you should add the domains in `/etc/hosts` file by command 
+   ```bash
+   bash -c 'echo "127.0.0.1 <your-dashboard-domain> <your-controller-domain> <your-radius-domain> <your-topology-domain>" >> /etc/hosts'
+   ```
+   - If you are using pipenv, remember that changing the values in `.env` file does nothing because `.env` is also a special file in `pipenv`, you need to change the values in `.env` file then re-activate environment to ensure that the changes reflect.
```

Comment:
I think this is confusing.  Since this is a docker project and all python environments are in containers, I don't understand why you would want to use Pipenv or any other virtual env.  It's not recomended as a step in the Docs.  Also, I don't think docker-compose would have an issue reading the .env file even if Pipenv is in use.

Reply:
Actually, I agree. I added this line when we were using that `openwisp-orchestration` container and it was relevant at that time but reading your point now, it doesn't make sense now because at lot of things have changed since the prototype.

So, @nikitaermishin can you please remove it! :smile:

Reply:
:+1: I fully expect you to remind me of some specific issue I had forgotten about. :smile:

Reply:
Ok, fixed it 😉